### DISCUSSION TITLE
Import/export DUH/IP-XACT

### DISCUSSION THREAD

**Discussion by olofk** [FPGAwars/icestudio#833] [Ideas]
Icestudio uses a custom format, but it would be very handy to be able to import existing components in IP-XACT or similar. A good target would perhaps be the [Duh](https://github.com/sifive/duh) format, which is pretty much the useful parts of IP-XACT described in JSON

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
